tutorials/Transfer_Learning_with_Megatron_for_Languages_Other_than_English.ipynb

```diff
@@ -0,0 +1,893 @@
+        "# get the list of supported BERT-like models, for the complete list of HugginFace models, see https://huggingface.co/models\n",
+        "print(nemo_nlp.modules.get_pretrained_lm_models_list(include_external=True))\n",
+        "\n",
+        "# specify BERT-like model, you want to use\n",
+        "#PRETRAINED_BERT_MODEL = \"megatron-bert-uncased\"\n",
+        "PRETRAINED_BERT_MODEL = \"bert-base-multilingual-uncased\"\n"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "RK2xglXyAUOO"
+      },
+      "source": [
+        "# add the specified above model parameters to the config\n",
+        "config.model.language_model.pretrained_model_name = PRETRAINED_BERT_MODEL"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "fzNZNAVRjDD-"
+      },
+      "source": [
+        "Now, we are ready to initialize our model. During the model initialization call, the dataset and data loaders we'll be prepared for training and evaluation.\n",
+        "Also, the pretrained BERT model will be downloaded, note it can take up to a few minutes depending on the size of the chosen BERT model."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "NgsGLydWo-6-"
+      },
+      "source": [
+        "model_from_scratch = nemo_nlp.models.TokenClassificationModel(cfg=config.model, trainer=trainer)"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "kQ592Tx4pzyB"
+      },
+      "source": [
+        "## Monitoring training progress\n",
+        "Optionally, you can create a Tensorboard visualization to monitor training progress."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "mTJr16_pp0aS"
+      },
+      "source": [
+        "try:\n",
+        "  from google import colab\n",
+        "  COLAB_ENV = True\n",
+        "except (ImportError, ModuleNotFoundError):\n",
+        "  COLAB_ENV = False\n",
+        "\n",
+        "# Load the TensorBoard notebook extension\n",
+        "if COLAB_ENV:\n",
+        "  %load_ext tensorboard\n",
+        "  %tensorboard --logdir {exp_dir}\n",
+        "else:\n",
+        "  print(\"To use tensorboard, please use this notebook in a Google Colab environment.\")"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "LqLZHfWX2ymY"
+      },
+      "source": [
+        "In case of using Megatron LM (megatron-bert-uncased and megatron-bert-cased), you will need to provide checkpoint, config file and vocab file. For getting checkpoints and config files, see: https://github.com/NVIDIA/Megatron-LM, https://ngc.nvidia.com/catalog/models/nvidia:megatron_bert_345m/files\n",
+        "\n",
```

Comment:
If we want to remove Megatron as we discussed, then these are not needed anymore, right? or you still want this text and the next code cells be there just in case someone decides to use Megatron? 
And I think we need to give all three of checkpoint, config_file, and vocab_file in case of using megatron-bert-uncased, that is assumed here

Reply:
Let's remove Megatron since this notebook talks about a non-Eng downstream task and Megatron was trained only on Eng data

Reply:
Okay.